basic/web-sockets/README.md

```diff
@@ -21,7 +21,7 @@ The `org.springframework.integration.samples.websocket.standard.client.Applicati
 ## Browser Client
    
 The `index.html` in the `src/main/resources/static` directory of this project demonstrates a JavaScript `SockJS` client, which connects
-to our server and just prints its messages in the middle of page.
+to our server and just prints its messages in the middle of page. It can be reached at http://localhost:8080/index.html.
```

Comment:
At first I tried opening the `index.html` file on http://localhost:<anotherport> but I kept getting CORS errors. Until I finally tried the url above which worked.